### PR TITLE
RFC: auto-fixes from ruff 0.6 (visualization)

### PR DESCRIPTION
**Pull request by neutrinoceros**
### Description
Partially address #16863 for `visualization`.

- [ ] By checking this box, the PR author has requested that maintainers do **NOT** use the "Squash and Merge" button. Maintainers should respect this when possible; however, the final decision is at the discretion of the maintainer that merges the PR.

### CODE REVIEW

File: astropy/visualization/lupton_rgb.py [hunk 1]
```diff
@@ -267,8 +267,7 @@ def __init__(self, minimum, stretch, Q=8):
             Q = 0.1
         else:
             Qmax = 1e10
-            if Q > Qmax:
-                Q = Qmax
+            Q = min(Q, Qmax)
```
Comment (astrofrog) [inline astropy/visualization/lupton_rgb.py:270]: Is Qmax even needed? (Could write 1e10 directly in min call?)

Comment (neutrinoceros) [reply astropy/visualization/lupton_rgb.py]: good call. Done !

**Review by mhvk — APPROVED** [review]
Also nice!

**Comment by larrybradley** [comment]
@neutrinoceros `pre-commit.ci` is failing due to a timeout, but I can't restart that job.

**Comment by pllim** [comment]
I don't think anyone can "restart" pre-commit job. It is not going through the Actions interface. Maybe try rebase?

**Comment by pllim** [comment]
So unlike the other sub-packages, performance impact isn't a worry here?

**Comment by neutrinoceros** [comment]
before I rebase, let me try this:
pre-commit.ci run

**Comment by pllim** [comment]
Great! CI is green now but I will leave the merging to sub-package maintainers. Thanks, all!

**Comment by mhvk** [comment]
> So unlike the other sub-packages, performance impact isn't a worry here?

I'm not sure it is in a critical path anywhere, just had not expected it to be worse performance in general.

**Pull request merged**